packages/frontend/src/commons/chains/base.ts

```diff
@@ -24,22 +24,22 @@ export const baseData: ChainData = {
             symbol: "WETH",
         },
         {
-            address: "0x833589fcd6edb6e08f4c7c32d4f71b54bda02913",
+            address: "0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913",
             decimals: 6,
             name: "USDC",
             symbol: "USDC",
         },
-        {
```

Comment:
We still need this, since it was added to the base tokens in the backend, right?